examples/tutorials/translations/german/Part 01 - The Basic Tools of Private Deep Learning.ipynb

```diff
@@ -0,0 +1,962 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Tutorial: Die grundlegenden Werkzeuge des privaten Deep Learning\n",
+    "\n",
+    "Willkommen zu PySyfts Einführungs-Tutorial zum Schutz der Privatsphäre und zum dezentralen Deep Learning. Diese Reihe von Notizbüchern ist eine schrittweise Anleitung, mit der Sie die neuen Tools und Techniken kennenlernen können, die für das gründliche Lernen geheimer / privater Daten / Modelle erforderlich sind, ohne sie unter einer Autorität zu zentralisieren.\n",
+    "\n",
+    "** Geltungsbereich: ** Beachten Sie, dass wir nicht nur über das Dezentralisieren / Verschlüsseln von Daten sprechen, sondern auch darüber, wie PySyft verwendet werden kann, um das gesamte Ökosystem um Daten herum zu dezentralisieren, einschließlich der Datenbanken, in denen sich Daten befinden gespeichert und abgefragt sowie die neuronalen Modelle, mit denen Informationen aus Daten extrahiert werden. Wenn neue Erweiterungen für PySyft erstellt werden, werden diese Notizbücher um neue Tutorials erweitert, um die neuen Funktionen zu erläutern.\n",
+    "\n",
+    "Autoren:\n",
+    "- Andrew Trask - Twitter: [@iamtrask](https://twitter.com/iamtrask)\n",
+    "\n",
+    "Übersetzer:\n",
+    "- Vineet Jain - Github: [@vineetjai](https://github.com/vineetjai)\n",
+    "\n",
+    "## Gliederung:\n",
+    "\n",
+    "- Teil 1: Die grundlegenden Werkzeuge des privaten Deep Learning\n",
+    "\n",
+    "\n",
+    "## Warum dieses Tutorial machen?\n",
+    "\n",
+    "** 1) Ein wettbewerbsfähiger Karrierevorteil ** - In den letzten 20 Jahren hat die digitale Revolution die Daten in immer größeren Mengen zugänglicher gemacht, da analoge Prozesse digitalisiert wurden. Mit neuen Regelungen wie z [GDPR](https://eugdpr.org/),Unternehmen stehen unter dem Druck, weniger Freiheit bei der Verwendung - und vor allem bei der Analyse - persönlicher Informationen zu haben. ** Fazit: ** Data Scientists werden mit \"Old School\" -Tools nicht auf so viele Daten zugreifen können, aber durch das Erlernen der Tools von Private Deep Learning können SIE dieser Kurve voraus sein und einen Wettbewerbsvorteil erzielen Deine Karriere.\n",
```

Comment:
- I had translated English tutorials as it is.

- I don't think "z" will be changed to "zB." because of "such as." in English doesn't exist.

- I had done a mistake in this one. As I am a beginner in German.